packages/docusaurus-cssnano-preset/src/remove-overridden-custom-properties/index.js

```diff
@@ -28,11 +28,13 @@ module.exports = postcss.plugin(
         const sameProperties =
           decl.parent.nodes.filter((n) => n.prop === decl.prop) || [];
         const hasImportantProperties = sameProperties.some((p) =>
-          p.hasOwnProperty('important'),
+          Object.prototype.hasOwnProperty.call(p, 'important'),
         );
 
         const overriddenProperties = hasImportantProperties
-          ? sameProperties.filter((p) => !p.hasOwnProperty('important'))
+          ? sameProperties.filter(
+              (p) => !Object.prototype.hasOwnProperty.call(p, 'important'),
+            )
```

Comment:
that feels a bit verbose, wonder if we shouldn't just disable this eslint rule instead?

Reply:
I think it's fair to disable it, as much as leaving it on. The side effects or DDoS issues mentioned [on the rule page](https://eslint.org/docs/rules/no-prototype-builtins) seems not be related to the way those methods are used in the Docusuaurs code, but in the other hand I think that the slightly more verbose syntax won't hurt anyone, and there are only a few usages of `hasOwnProperty` in the wholde codebase.